src/main/java/com/revature/models/CurriculumTopic.java

```diff
@@ -28,70 +30,88 @@
 	@GeneratedValue(strategy=GenerationType.SEQUENCE, generator="curriculum_topics_seq_name")
 	private Integer id;
 	
-//	@ManyToOne
-//	@JoinColumn(name="curriculum_id", nullable=false, referencedColumnName="id")
+	@ManyToOne(fetch = FetchType.EAGER, optional= false, cascade =CascadeType.ALL)
+	@JoinColumn(name="curriculum_id", nullable=false, referencedColumnName="id")
 	@NotNull
-	@Column(name="curriculum_id")
-	private Integer curriculumId;
+	private Curriculum curriculumId;
 	
-//	@ManyToOne
-//	@JoinColumn(name="topic_id", nullable=false, referencedColumnName="id")
+	@ManyToOne(fetch = FetchType.EAGER, optional= false, cascade =CascadeType.ALL)
+	@JoinColumn(name="topic_id", nullable=false, referencedColumnName="id")
 	@NotNull
-	@Column(name="topic_id")
-	private Integer topicId;
+	private Topic topicId;
 	
-//	@NotNull
-	@Column(name="number_of_weeks")
+	@NotNull
+	@Column(name="week_number")
 	private Integer numberOfWeeks;
 	
 	public CurriculumTopic() {}
 
-	public CurriculumTopic(Integer curriculum_id, Integer topic_id, Integer number_of_weeks) {
+	
+
+	public CurriculumTopic(Curriculum curriculumId, Topic topicId, Integer numberOfWeeks) {
 		super();
-		this.curriculumId = curriculum_id;
-		this.topicId = topic_id;
-		this.numberOfWeeks = number_of_weeks;
+		this.curriculumId = curriculumId;
+		this.topicId = topicId;
+		this.numberOfWeeks = numberOfWeeks;
 	}
 
-	public CurriculumTopic(Integer id, Integer curriculum_id, Integer topic_id, Integer number_of_weeks) {
+	public CurriculumTopic(Integer id, Curriculum curriculumId, Topic topicId, Integer numberOfWeeks) {
 		super();
 		this.id = id;
-		this.curriculumId = curriculum_id;
-		this.topicId = topic_id;
-		this.numberOfWeeks = number_of_weeks;
+		this.curriculumId = curriculumId;
+		this.topicId = topicId;
+		this.numberOfWeeks = numberOfWeeks;
 	}
 
+
+
 	public Integer getId() {
 		return id;
 	}
 
+
+
 	public void setId(Integer id) {
 		this.id = id;
 	}
 
-	public Integer getCurriculum_id() {
+
+
+	public Curriculum getCurriculumId() {
 		return curriculumId;
 	}
 
-	public void setCurriculum_id(Integer curriculum_id) {
-		this.curriculumId = curriculum_id;
+
+
+	public void setCurriculumId(Curriculum curriculumId) {
+		this.curriculumId = curriculumId;
 	}
 
-	public Integer getTopic_id() {
+
+
+	public Topic getTopicId() {
 		return topicId;
 	}
 
-	public void setTopic_id(Integer topic_id) {
-		this.topicId = topic_id;
+
+
+	public void setTopicId(Topic topicId) {
+		this.topicId = topicId;
 	}
 
-	public Integer getNumber_of_weeks() {
+
+
+	public Integer getNumberOfWeeks() {
 		return numberOfWeeks;
 	}
 
-	public void setNumber_of_weeks(Integer number_of_weeks) {
-		this.numberOfWeeks = number_of_weeks;
+
+
+	public void setNumberOfWeeks(Integer numberOfWeeks) {
+		this.numberOfWeeks = numberOfWeeks;
 	}
+
+
```

Comment:
be sure to have only one empty line space between methods. good eye on the snake casing